pkg/service/backup/service.go

```diff
@@ -807,17 +807,21 @@ func (s *Service) Backup(ctx context.Context, clusterID, taskID, runID uuid.UUID
 		return nil
 	}
 	stageFunc := map[Stage]func() error{
-		StageAwaitSchema: func() error {
+		StageAwaitSchema: func() error { // nolint: unparam
```

Comment:
Why `nolint: unparam` ?

Reply:
Otherwise we get the following error while running `make check`:
```
pkg/service/backup/service.go:810:28: (*Service).Backup$3 - result 0 (error) is always nil (unparam)
                StageAwaitSchema: func() error {
                                         ^
```